CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * BraintreePayPalNativeCheckout (General Availability release)
   * Update PayPalCheckout from 0.110.0 to 1.0.0. This is our newly released General Availability version
      * _Note: This module will now be subject to semantic versioning_
+  * Fixed a bug where post vaulting FirstName, LastName and email are null but a nonce is generated.
```

Comment:
We will want to move this into a new `## unreleased` section since this section was released in [version 6.3.0](https://github.com/braintree/braintree_ios/blob/main/CHANGELOG.md#630-2023-07-10). Please ensure you've merged the latest from main into this branch to pull in the updated changelog from the version that was released.